README.md

```diff
@@ -14,8 +14,57 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the form of environment variables. They 
+may be a bit different between packages, but the most common are ``BRANCH``,
+``VERSION`` and ``RELEASE``.
+
+So in order to build a specific branch / version, this command can be used from
+the folder of the package we want to build:
+
+    make BRANCH=qa/1.x VERSION=1.7.0 RELEASE=rc5
+
+Keep in mind that the Makefiles are a bit recursive, they will invoke docker, 
+mount the current folder, and run the deb-build target.
+
+#### Development
+
+Some makefiles have a "dev" target, that give you a shell inside of the docker 
+container used to build packages. When you are inside the container, the command needed to
+build the packages is:
+
+    make deb-build
+
+### Repositories management
+
+This repo generates Archivematica packages, but also packages that need to be 
+installed in order for Archivematica to run. They are placed in the 
```

Comment:
L41 and L42 have trailing whitespace.

![image](https://user-images.githubusercontent.com/606459/38200561-b4a52fc6-3649-11e8-8298-58d062825397.png)

It also happens in L19, L28, L33...

If you're using Sublime, see `trim_trailing_white_space_on_save`.
There is a similar solution in other editors. In `.vimrc` you can show them with:

```
set list
set listchars=tab:»·,trail:·,nbsp:%,extends:>,precedes:<
```